source/frontend/js/calorie-accountant.js

```diff
@@ -1,17 +1,22 @@
 angular.module('calorieAccountant', ['ui.router']) // Defines the application (no dependencies injected)
   .config(['$stateProvider', function($stateProvider) {
-    $stateProvider.state('home', {
-      url: '/',
-      templateUrl: 'templates/home.html',
-      controller: 'indexCtrl as ctrl'
+    $stateProvider
```

Comment:
I prefer this "fluent" approach.

http://en.wikipedia.org/wiki/Fluent_interface